src/components/SideBar/styled.ts

```diff
@@ -43,4 +38,116 @@ export const StyledSideBar = styled('aside')<StyledSideBarProps>`
             font-size: 80%;
         }
     }
+    //
+`;
+
+// export const flagStyle = { '& > img': { mr: 2, flexShrink: 0 } };
+
+export const FlagIcon = styled('img')(() => ({
+    marginRight: '2rem',
+    flexShrink: 0,
+}));
+
+export const SideBarHeader = styled('div')(({ theme }) => ({
+    cursor: 'pointer',
+    height: 'auto',
+    position: 'relative',
+    h1: {
+        borderRadius: '4px',
+        display: 'inline-block',
+        fontSize: '12px',
+        margin: 0,
+        padding: '0.7ex 2ex',
+        background: theme.primary.main,
+        color: theme.primary.contrastText,
+    },
+}));
+
+export const LatestGlobal = styled('aside')<{ $sidebaropen: boolean }>`
+    margin: 2ex 0;
+    #total-cases,
+    #total-deaths,
+    #p1-cases,
+    #b1351-cases {
+        display: none;
+        color: black;
+        font-size: 2.7rem;
+        font-weight: 500;
+        margin-right: 0.35ex;
+    }
+    #total-cases {
+        display: inline;
+    }
+    .last-updated-date {
+        font-size: 1.2rem;
+        color: #999;
+        margin-top: 2rem;
+    }
+`;
+
+export const SearchBar = styled('div')`
+    margin-top: 0;
+    div:first-of-type {
+        width: 100%;
+    }
+    .MuiAutocomplete-endAdornment {
+        display: flex;
+        flex-flow: row-reverse;
+    }
+    .MuiAutocomplete-popper {
+        button {
+            background-color: #fff;
+        }
+    }
+    .autocompleteBox {
```

Comment:
I believe this is not relevant anymore since you used `FlagIcon` object to style this. If this is the case then delete this part